src/test/test.js

```diff
@@ -592,6 +592,36 @@ describe ('Tests based on standard JSON Schema Test Suite', () => {
       return schema;
     });
 
+		test('format', 'validation of date-time strings', () => {
```

Comment:
Nice work, @psvet! Since these tests aren't part of the standard json schema test suite, maybe at least wrap inside of a describe() block to indicate that they are extra tests...